apps/heureka/src/components/Services/ServicePanel/ServicePanel.tsx

```diff
@@ -0,0 +1,181 @@
+            <DataGridHeadCell>Version</DataGridHeadCell>
+            <DataGridHeadCell>Critical</DataGridHeadCell>
+            <DataGridHeadCell>High</DataGridHeadCell>
+            <DataGridHeadCell>Medium</DataGridHeadCell>
+            <DataGridHeadCell>Low</DataGridHeadCell>
+            <DataGridHeadCell>Actions</DataGridHeadCell>
+          </DataGridRow>
+          {loading ? (
+            <EmptyDataGridRow colSpan={7}>Loading...</EmptyDataGridRow>
+          ) : formattedImageVersions?.length === 0 ? (
+            <EmptyDataGridRow colSpan={7}>No image versions available.</EmptyDataGridRow>
+          ) : error ? (
+            <EmptyDataGridRow colSpan={7}>
+              <Message variant="error">{error}</Message>
+            </EmptyDataGridRow>
+          ) : (
+            formattedImageVersions.map((version, index) => (
+              <DataGridRow key={index}>
+                <DataGridCell className="break-all overflow-hidden">
+                  <Stack gap="1" direction="vertical">
+                    <span>{version.imageName}</span>
+                    <Stack gap="1" alignment="center">
+                      <a
+                        href={`https://${version.imageName}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="hover:underline text-sm"
+                      >
+                        <Stack gap="1.5" alignment="center">
+                          <Icon icon="openInNew" size="16" color="jn-global-text" onClick={() => {}} />
+                          <span>Image repository</span>
+                        </Stack>
+                      </a>
+                    </Stack>
+                  </Stack>
+                </DataGridCell>
+                <DataGridCell>
+                  <Tooltip triggerEvent="hover" placement="top">
+                    <TooltipTrigger>{truncateVersion(version.imageVersion)}</TooltipTrigger>
+                    <TooltipContent>{version.imageVersion}</TooltipContent>
+                  </Tooltip>
+                </DataGridCell>
+                <DataGridCell>
+                  {version.issueCounts.critical ? (
+                    <Badge icon text={version.issueCounts.critical.toString()} variant="danger" />
+                  ) : (
+                    "-"
+                  )}
+                </DataGridCell>
+                <DataGridCell>
+                  {version.issueCounts.high ? (
+                    <Badge icon text={version.issueCounts.high.toString()} variant="warning" />
+                  ) : (
+                    "-"
+                  )}
+                </DataGridCell>
+                <DataGridCell>{version.issueCounts.medium || "-"}</DataGridCell>
+                <DataGridCell>{version.issueCounts.low || "-"}</DataGridCell>
+                <DataGridCell>
+                  <Button
+                    disabled
+                    variant="primary"
```

Comment:
Primary buttons are like Highlanders, there can only be one on each page ;) 
So in this case where you map over many items and render a button for each do not use `variant="primary"` but instead just use the default variant for these cases